contact.html

```diff
@@ -21,55 +21,25 @@
     <!-- MAIN CSS -->
     <link rel="stylesheet" href="css/style.css">
 
+    <!-- Google tag (gtag.js) -->
+    <script defer src="https://www.googletagmanager.com/gtag/js?id=G-C36HVXWMSM"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'G-C36HVXWMSM');
+    </script>
+    <script src="js/components/header.js" type="text/javascript" defer></script>
+    <script src="js/components/footer.js" type="text/javascript" defer></script>
```

Comment:
we are only using the 5 pages in the nav, so no need to update the header of the other pages. Fine to leave it but next time lets just skip them. I probably should delete them but I was keeping them around in case I wanted to use some of their html in the other pages.